exercises/practice/perfect-numbers/.meta/config.json

```
@@ -3,20 +3,7 @@
     "sbromberger"
   ],
   "contributors": [
-    "alebaffa",
-    "bitfield",
-    "ekingery",
     "ferhatelmas",
```

```suggestion
    "ferhatelmas"
```
I think you have to remove that , to make please json.